src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1576,20 +1601,20 @@ module Mutations = struct
       |> Result.of_option ~error:"Invalid signature"
       |> Deferred.return
     in
-    send_user_command coda command
+    Deferred.return @@ send_user_command coda command
```

Comment:
could you use `let%map` above, so you don't need this `return`?

Reply:
We are in a Deferred.Result monad, and `send_user_command` would return a value of Result monad. I am using Deferred.return to lift it up. I don't think there's another way to do it.